### PR TITLE
Add DebugBundle config reference

### PR DESCRIPTION
**Pull request by wouterj**
After this PR, the components still contain some information about full-stack, which we should move to a cookbook recipe imo.

| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.6+ |
| Fixed tickets | - |

### CODE REVIEW

File: components/var_dumper/introduction.rst [hunk 1]
```diff
@@ -70,14 +71,14 @@ current PHP SAPI:
...
+    #. From time to time, run ``composer global update symfony/var-dumper``
+       to have the latest bug fixes.
 
 DebugBundle and Twig Integration
 --------------------------------
 
-The ``DebugBundle`` allows greater integration of the component into the
-Symfony full stack framework. It is enabled by default in the *dev* and *test*
+The DebugBundle allows greater integration of the component into the Symfony
+full stack framework. It is enabled by default in the *dev* and *test*
```
Comment (xabbuh) [inline components/var_dumper/introduction.rst]: full-stack

**Comment by WouterJ** [comment]
Thanks @xabbuh! I've fixed your comment

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thanks Wouter. I made some minor tweaks in 9231f92 and opened #5754 to move bundle related docs from the component section to the cookbook.

**Pull request closed without merging**